docs/apache-airflow-providers-common-sql/index.rst

```diff
@@ -18,6 +18,20 @@
 ``apache-airflow-providers-common-sql``
 =======================================
 
+Supported Database Types
+------------------------
+
+The ``apache-airflow-providers-common-sql`` provider is compatible with the following
+database types:
+
+- :doc:`PostgreSQL <apache-airflow-providers-postgres:index>`
```

Comment:
Does this hyperlink works?

Reply:
Yep it works, I'm also had a filling that it might not work correctly

![image](https://github.com/apache/airflow/assets/3998685/819639a9-2541-4bfe-afff-14369e51a8c6)

I'm not sure is should be in to of the index, maybe separate page? In general looks good but lets see what the other say

Reply:
Yes. Agree with @Taragolis - I think it makes sense to Add it as a separate page, then in the future it will be easier to automate it, because eventually we want this page to be generated automatically. This should be as easy as adding a new .rst  file and add it to the Index (above the "the remainder of this page is generated automatically).

One more thing - I think it should be a much more complete list as well:

From the slack thread, this the complete list:

Generated with:

```
find . -name index.rst | grep -v '_api' | xargs grep -l common.sql
```

```
/docs/apache-airflow-providers-postgres/index.rst
./docs/apache-airflow-providers-apache-impala/index.rst
./docs/apache-airflow-providers-elasticsearch/index.rst
./docs/apache-airflow-providers-microsoft-mssql/index.rst
./docs/apache-airflow-providers-slack/index.rst
./docs/apache-airflow-providers-databricks/index.rst
./docs/apache-airflow-providers-vertica/index.rst
./docs/apache-airflow-providers-jdbc/index.rst
./docs/apache-airflow-providers-apache-hive/index.rst
./docs/apache-airflow-providers-apache-druid/index.rst
./docs/apache-airflow-providers-mysql/index.rst
./docs/apache-airflow-providers-odbc/index.rst
./docs/apache-airflow-providers-snowflake/index.rst
./docs/apache-airflow-providers-amazon/index.rst
./docs/apache-airflow-providers-common-sql/index.rst
./docs/apache-airflow-providers-presto/index.rst
./docs/apache-airflow-providers-trino/index.rst
./docs/apache-airflow-providers-apache-pinot/index.rst
./docs/apache-airflow-providers-pgvector/index.rst
./docs/apache-airflow-providers-sqlite/index.rst
./docs/apache-airflow-providers-exasol/index.rst
./docs/apache-airflow-providers-google/index.rst
./docs/apache-airflow-providers-openlineage/index.rst
./docs/apache-airflow-providers-apache-drill/index.rst
```

Reply:
I created another file inside `/apache-airflow-provders-common-sql`. Is this a right approach? I'm not familiar with Sphinx. 

When I build this doc, I got many errors like 
```
############################## Start docs build errors summary ##############################

============================== apache-airflow-providers-common-sql ==============================
------------------------------ Error   1 --------------------
 WARNING: toctree contains reference to nonexisting document '_api/airflow/providers/common/sql/index'

File path: apache-airflow-providers-common-sql/index.rst (43)

  38 |     :caption: Guides
  39 | 
  40 |     Connecting to SQL Databases <connections>
  41 |     Operators <operators>
  42 | 
> 43 | .. toctree::
  44 |     :hidden:
  45 |     :maxdepth: 1
  46 |     :caption: References
  47 | 
  48 |     Python API <_api/airflow/providers/common/sql/index>
------------------------------ Error   2 --------------------
 WARNING: toctree contains reference to nonexisting document '_api/airflow/providers/common/sql/index'

File path: apache-airflow-providers-common-sql/index.rst (43)

  38 |     :caption: Guides
  39 | 
  40 |     Connecting to SQL Databases <connections>
  41 |     Operators <operators>
  42 | 
> 43 | .. toctree::
  44 |     :hidden:
  45 |     :maxdepth: 1
  46 |     :caption: References
  47 | 
  48 |     Python API <_api/airflow/providers/common/sql/index>
------------------------------ Error   3 --------------------
 WARNING: toctree contains reference to nonexisting document '_api/airflow/providers/common/sql/index'

File path: apache-airflow-providers-common-sql/index.rst (43)

  38 |     :caption: Guides
  39 | 
  40 |     Connecting to SQL Databases <connections>
  41 |     Operators <operators>
  42 | 
> 43 | .. toctree::
  44 |     :hidden:
  45 |     :maxdepth: 1
  46 |     :caption: References
  47 | 
  48 |     Python API <_api/airflow/providers/common/sql/index>
```

Reply:
Try with `--clean-build` flag - this happens when there is a build error and some old stuff remains.

Reply:
I tried with the flag and now I got one error/warning like this
```
############################## Start docs build errors summary ##############################

============================== apache-airflow-providers-common-sql ==============================
------------------------------ Error   1 --------------------
/opt/airflow/docs/apache-airflow-providers-common-sql/supported-database-types.rst: WARNING: document isn't included in any toctree


############################## End docs build errors summary ##############################


The documentation has errors.
####################  Packages errors summary  ####################
Package name                                    Count of doc build errors    Count of spelling errors
```

Reply:
Yep. It should be added to the index.rst - similarly as other files

Reply:
@potiuk  I just made a folder for the new file. Does this look fine to you, or should I make the folder somewhere else?